java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -7787,5 +7787,4 @@ public TaskomaticApi getTaskomaticApi() {
     private static List<String> maybeActivationKeys(String activationKey) {
         return StringUtils.isEmpty(activationKey) ? Collections.emptyList() : List.of(activationKey);
     }
-
```

Comment:
This change seems unrelated to the patch and could probably be omitted (= 1 file less that is changed).

Reply:
Reset to the original version.